package_linter.py

```diff
@@ -132,7 +132,8 @@ def check_script(path, script_name):
     if check_file_exist(script_path) == 0:
         return
     print (c.BOLD + c.HEADER + "\n>>>>",
-           scripts[i].upper(), "SCRIPT <<<<" + c.END)
+           script_name.upper(), "SCRIPT <<<<" + c.END)
```

Comment:
It wasn't for me, I got an error there, you probably made some modifications in the mean time after it was working for you.